### PR TITLE
Add support for chatgpt-4o-latest

### PR DESCRIPTION
**Pull request by Congyuwang**

### CODE REVIEW

File: tiktoken-rs/src/tokenizer.rs [hunk 1]
```diff
@@ -30,6 +30,7 @@ pub enum Tokenizer {
 
 const MODEL_PREFIX_TO_TOKENIZER: &[(&str, Tokenizer)] = &[
     // chat
+    ("chatgpt-4o-latest", Tokenizer::O200kBase),
```
Comment (zurawiki) [inline tiktoken-rs/src/tokenizer.rs:33]: Since this is a specific model and not a model prefix. Let's add it to `MODEL_TO_TOKENIZER`.

**Review by zurawiki — CHANGES_REQUESTED** [review]
Make sure to add a test here: https://github.com/zurawiki/tiktoken-rs/blob/784f0e53990da982046128d08400f6ba4557fc7f/tiktoken-rs/src/tokenizer.rs#L144

**Comment by zurawiki** [comment]
Thanks for the PR. This LGTM after addressing my comments

**Comment by Congyuwang** [comment]
I've updated the PR and also added a test.

**Pull request merged**